### PR TITLE
PAYARA-3703 Update-file-users command now works when there are instances with different configs

### PR DESCRIPTION
**Pull request by Cousjava**
This is effectively reverting 0ef96b5168b4692a8e8a3816f5ba885ab0c1c31c

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: nucleus/security/core/src/main/java/com/sun/enterprise/security/cli/CreateFileUser.java [hunk 1]
```diff
@@ -95,7 +95,7 @@
 @Service(name = "create-file-user")
 @PerLookup
 @I18n("create.file.user")
-@ExecuteOn({ RuntimeType.ALL })
+@ExecuteOn({ RuntimeType.INSTANCE, RuntimeType.DAS })
 @TargetType({ DAS, STANDALONE_INSTANCE, CLUSTER, CONFIG })
```
Comment (smillidge) [inline nucleus/security/core/src/main/java/com/sun/enterprise/security/cli/CreateFileUser.java]: Should have deployment group as a target

File: nucleus/security/core/src/main/java/com/sun/enterprise/security/cli/DeleteFileUser.java [hunk 2]
```diff
@@ -90,7 +90,7 @@
 @Service(name = "delete-file-user")
 @PerLookup
 @I18n("delete.file.user")
-@ExecuteOn({ RuntimeType.ALL })
+@ExecuteOn({ RuntimeType.DAS, RuntimeType.INSTANCE })
 @TargetType({ CommandTarget.DAS, CommandTarget.STANDALONE_INSTANCE, CommandTarget.CLUSTER, CommandTarget.CONFIG })
```
Comment (smillidge) [inline nucleus/security/core/src/main/java/com/sun/enterprise/security/cli/DeleteFileUser.java]: Should have deployment group as a target type

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**